backend/app/adapter/sqldb/short_link_integration_test.go

```diff
@@ -724,6 +724,90 @@ func TestShortLinkSql_GetShortLinkByAliases(t *testing.T) {
 	}
 }
 
+func TestShortLinkSql_DeleteShortLink(t *testing.T) {
+	var (
+		twoYearsAgo = mustParseTime(t, "2018-05-01T08:02:16-07:00")
+		now         = mustParseTime(t, "2020-05-01T08:02:16-07:00")
```

Comment:
```suggestion
		createdAt = mustParseTime(t, "2018-05-01T08:02:16-07:00")
		expiredAt         = mustParseTime(t, "2020-05-01T08:02:16-07:00")
```
The variable names will not be true in the future